projects/ad4696_fmc/common/ad4696_bd.tcl

```diff
@@ -0,0 +1,123 @@
+
+create_bd_intf_port -mode Master -vlnv analog.com:interface:spi_master_rtl:1.0 ad4696_spi
+
+## To support the 1MSPS (SCLK == 80 MHz), set the spi clock to 160 MHz
+
+ad_ip_instance axi_clkgen spi_clkgen
+ad_ip_parameter spi_clkgen CONFIG.CLK0_DIV 5
+ad_ip_parameter spi_clkgen CONFIG.VCO_DIV 1
+ad_ip_parameter spi_clkgen CONFIG.VCO_MUL 8
+ad_connect spi_clkgen/clk sys_ps7/FCLK_CLK0
```

Comment:
It's not a good idea to use a main clock source directly. All the main clock sources have a generic net assigned to them, e.g. sys_cpu_clk. This way we can keep the board design file platform-independent.

Reply:
yes i remember you also sugested this fix for the ad4630. will fix this.